less/mosaico/static-pages/fragor-och-svar.less

```diff
@@ -42,7 +42,7 @@
 
   .faq__answer {
     display: none;
-    position: absolute;
+    position: relative;
```

Comment:
Can this line maybe be removed, if it doesn't change anything?

Reply:
Yes, it had been changed and therefore was made redundant, but I forgot to remove it. Have fixed it in the next PR.